README.md

```diff
@@ -2,121 +2,16 @@
-The input to use is managed by the [mux](http://wiki.ros.org/topic_tools/mux) node. It can be changed through a rosservice call such as,
-```terminal
-rosservice call /mux/select "topic: 'joy_vel'"
-```
-
-### key
-The `key` app allows the keyboard to control the robot.
-See also the `teleop` application for further details.
-
-To start the app, simply type the following command,
-```terminal
-turtlebot3c.key
-```
-This command automatically select 'key_vel' as the input for the [mux](http://wiki.ros.org/topic_tools/mux) node.
-
-### joy
-The `joy` app allows the joystick to control the robot.
-See also the `teleop` application for further details.
-
-To start the app, simply type the following command,
-```terminal
-turtlebot3c.joy
-```
-This command automatically select 'joy_vel' as the input for the [mux](http://wiki.ros.org/topic_tools/mux) node.
-
-### mapping
-The `mapping` app is a daemon that allows the robot to build a representation of its environment as a map usable by the navigation stack.
-To launch the mapping, simply type the following command:
-```terminal
-sudo snap start turtlebot3c.mapping
-```
-Drive your robot around in order to map your environment.
-Notice that you can monitor the process through Rviz.
-Once the entire environment covered, stop the mapping by typing the following command:
-```terminal
-sudo snap stop turtlebot3c.mapping
-```
-This will stop the process and automatically save the map as
-`$SNAP_USER_COMMON/map/${DATE}.yaml`.
-Furthermore, a softlink to the newly created map is created at
-`$SNAP_USER_COMMON/map/current_map.yaml`. The `navigation` stack will automatically use the softlinked map.
-This command automatically select 'key_vel' as the input for the [mux](http://wiki.ros.org/topic_tools/mux) node.
-
-Note that the default `USER` of snap daemons is `root`. Hence, the `$SNAP_USER_COMMON` variable will point to `/root/snap/turtlebot3c/common`.
-
-### navigation
-The `navigation` app is also a daemon that encompasses the whole autonomous navigation stack.
-It allows the robot to move autonomously in its environment, from its current location to a desired goal while avoiding obstacles.
-To start the navigation app, simply type the following command:
-```terminal
-sudo snap start turtlebot3c.navigation
-```
-It will start the whole navigation stack and use the last map built with the
-`mapping` app found at `$SNAP_USER_COMMON/map`.
-This command automatically select 'nav_vel' as the input for the [mux](http://wiki.ros.org/topic_tools/mux) node and switch back to 'key_vel' on exit.
-You can stop it by typing:
-```terminal
-sudo snap stop turtlebot3c.navigation
-```
+The project is composed of three snaps,
 
+- [`turtlebot3c-bringup`](./turtlebot3c-bringup-snap/README.md) is the core component of the robot as it handles all the drivers, sensors, basic ROS nodes necessary for a functioning robot.
+- [`turtlebot3c-teleop`](./turtlebot3c-teleop-snap/README.md) allows for teleoperating the robot.
```

Comment:
"allows for teleoperating the robot." sound weird to me.
Maybe something like: "allows teleoperation of the robot"